modules/preview-react/side-panel/lib/SidePanel.tsx

```diff
@@ -33,7 +38,7 @@ export interface SidePanelProps {
    *
    * @default 'left'
    */
-  origin?: 'left' | 'right';
+  origin?: SidePanelOrigin;
```

Comment:
let's keep this as a string union, when using it, the intellisense says left or right, rather than having the abstraction of the type, inlining the string union is a little nicer.
```suggestion
  origin?: 'left' | 'right';
```